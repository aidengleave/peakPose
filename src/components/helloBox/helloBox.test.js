import React from "react";
import ReactDOM from "react-dom";
import HelloBox from "./index.js";
import { mount, shallow, render } from "enzyme";

describe("HelloBox", () => {
  it("should render correctly with no props", () => {
    const component = shallow(<HelloBox />);

    expect(component).toMatchSnapshot();
  });
});
