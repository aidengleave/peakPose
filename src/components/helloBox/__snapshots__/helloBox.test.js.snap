// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelloBox should render correctly with no props 1`] = `
<h1>
  Hello world!
</h1>
`;
